src/data/items.txt

```diff
@@ -11071,6 +11071,9 @@
 11043
 11044	packaged model train set	494254899	modeltrain_box.gif	usable	t	0
 11045	model train set	959599372	modeltrain.gif	usable		0
+11046	Crimbo training manual	990145553	trainmanual.gif	usable		0
```

Comment:
It's reusable right?

Reply:
yeah. And it might be curse-like, but I don't know what the page is.